storage/Utils/Logger.cc

```diff
@@ -20,53 +20,49 @@
  * find current contact information at www.novell.com.
  */
 
-
+#include <fcntl.h>
```

Comment:
Do we really need that header file? This is so low-level I can't remember when I last needed it.

Reply:
Leftover, I added it for the constants that we are replaced by octal numbers.